data/human/news.txt

```diff
@@ -851,6 +941,7 @@ news "athlete"
 			"I'm hoping to get a sponsorship, but nobody here wants to partner with me. Could you recommend a planet where I might find better results?"
 			"I don't expect everyone to be as fit as I am, but do remember to get up and exercise a little every once in a while!"
 			"I once went to a sports event on Dancer in the Rastaban system, but never again! All activities were indoors because of the horrible weather the planet has."
+			"You want to sponsor me? I've never lost a race! How many I've won? Well, not that many yet either..."
```

Comment:
Technically this would be "have I"; it depends on how excited this person is. Your choice.
```suggestion
			"You want to sponsor me? I've never lost a race! How many have I won? Well, not that many yet either..."
```